### ISSUE TITLE
Audit Issue with "marked" - Regular Expression Denial of Service

### ISSUE THREAD

**Issue by svewag** [pocka/storybook-addon-vue-info#96]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ marked                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.6.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ storybook-addon-vue-info [dev]                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ storybook-addon-vue-info > marked                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/812                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by pocka** [comment]
Solved at v1.1.2

**Comment by svewag** [comment]
Thank you
